### ISSUE TITLE
Config "dropPlayerHead" not working

### ISSUE THREAD

**Issue by JustCallMeAz** [B1n-ry/Youre-in-grave-danger#158]
Hi @B1n-ry , the config for the following doesn't work :
`"dropPlayerHead": false,  // Should the player drop their head? true/false`

Configured to false or true, it does always drop the player head.

Instance : 
`Java 1.20.1`

Mod launcher :
`Modrinth`

Mod version :
`2.0.0-beta.11`

If asked, I can provide any more informations.

**Comment by B1n-ry** [comment]
This is not the case for me. Do note that the head on the gravestone (if that's enabled) does not mean that it contains that player's head. You can configure rendering in the renderConfig section.
There's also a chance there's a mod present that will add the player head drop, which could in that case be the source of the issue.
Let me know if you get it working, have more questions, or more information

**Comment by JustCallMeAz** [comment]
You are right, I'm confused, I'm the one to blame for this misinvestigation.
It were caused by another mod which I have disabled, then I try the config `dropPlayerHead` of your mod and it's working as intended either true or false.

As it's my fault, I let you close or delete this github issue.

No more questions at the moment, what you are doing is great !
Might not be the appropriate place, but if there anyway I can contribute to what you are doing !
